software/apps/audio_module/event_detector/README.md

```diff
@@ -0,0 +1,14 @@
+# Fixed-point ridge tracker for acoustic event detection in C
```

Comment:
Your guess is correct! I've updated the README according to your suggestion. Thank you.